NEWS.md

```diff
@@ -76,10 +82,20 @@ Note: We're only listing outstanding class updates.
 
     * `Math.log1p` and `Math.expm1` are added. [[Feature #21527]]
 
-* Socket
+* Method
 
-    * `Socket.tcp` & `TCPSocket.new` accepts `open_timeout` as a keyword argument to specify
-      the timeout for the initial connection. [[Feature #21347]]
+    * `Method#source_location`, `Proc#source_location`, and
+      `UnboundMethod#source_location` now return extended location
+      information with 5 elements: `[path, start_line, start_column,
+      end_line, end_column]`. The previous 2-element format `[path,
+      line]` can still be obtained by calling `#take(2)` on the result.
+      [[Feature #6012]]
+
+* Proc
+
+    * `Proc#parameters` now shows anonymous optional parameters as `[:opt]`
+      instead of `[:opt, nil]`, making the output consistent across all
+      parameter types. [[Bug #20974]]
```

Comment:
```suggestion
    * `Proc#parameters` now shows anonymous optional parameters as `[:opt]`
      instead of `[:opt, nil]`, making the output consistent with when the
      anonymous parameter is required. [[Bug #20974]]
```